src/pages/UpdateUser.tsx

```diff
@@ -26,42 +28,79 @@ export const UpdateUser = () => {
     history.replace("/")
   })
 
+  const updateImage = useAsync(async (data: FormData) => {
+    const response = await api.uploadImage(data)
+    if (!response) return
+  })
+
+  const handleUpload = async (event: ChangeEvent<HTMLInputElement>) => {
+    const file = event.currentTarget.files as unknown as File[]
```

Comment:
```suggestion
    const files = event.currentTarget.files as unknown as File[]
```
số nhiều thì để biến dạng plural đi e
event.currentTarget.files ban đầu nó là type gì mà e phải cast vậy

Reply:
dạ nó ghi là trả về một FileList  | null em tưởng nó là object nên em cast em sửa rồi ạ.